src/data/thermal-conductivity-constants.ts

```diff
@@ -16,8 +16,402 @@ export const DIPPRThermalConductivityDictionary: Record<
     thermalConductivityAtMinimumTemperature: 0.2524,
     thermalConductivityAtMaximumTemperature: 0.1727,
   },
-};
 
+  Acetamide: {
```

Comment:
Again thermal conductivity data is being pushed into the PR.

Reply:
ok I will sync the local repo with remote repo and the will update  as said

Reply:
If you need help we can connect.